README.md

```diff
@@ -76,14 +82,20 @@ terraform workspace select <your_workspace>
 terraform destroy -var-file=<your_workspace>.yml
 ```
 
+## Notes ##
+
+Running `pre-commit` requires running `terraform init` in every directory that
+contains Terraform code. In this repository, these are the main directory and
```

Comment:
I think in this case this should read, "there are only the main directory."

Reply:
Updated in https://github.com/cisagov/assessment-data-import-terraform/pull/4/commits/2bab763073ca17d6f43ebbcb5910254a47647ed8